apps/condo/lang/en/en.json

```diff
@@ -1764,6 +1764,7 @@
   "meterReadingSource.Messenger.name": "Messengers (tg, whatsapp, viber, ..)",
   "meterReadingSource.SocialNetwork.name": "Social media (vk, fb, ..)",
   "meterReadingSource.MobileAppStaff.name": "Staff mobile app",
+  "meterReadingSource.metersBillingIntegrationKvp24.name": "Integration with kvp24",
```

Comment:
it is best to read it dynamically from connected b2b app

Reply:
Just renamed to "External integration".
But what do you mean?